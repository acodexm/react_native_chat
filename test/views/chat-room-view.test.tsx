import React from 'react';
import MockedComponent from '../support/mocked-component';
import ChatRoomView from '@views/chat/chat-room-view/chat-room-view';
import { render } from '@testing-library/react-native';

it('renders correctly', () => {
  const tree = render(<MockedComponent Component={ChatRoomView} />).toJSON();
  expect(tree).toMatchSnapshot();
});
