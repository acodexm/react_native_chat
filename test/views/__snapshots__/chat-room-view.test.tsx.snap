// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#ffffff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
        "paddingTop": 15,
      },
    ]
  }
/>
`;
